starter-code/pizza.js

```diff
@@ -65,21 +93,57 @@ function renderButtons() {
 
 function renderPrice() {
   // Iteration 4: change the HTML of `<aside class="panel price">`
+  /* state.forEach((elem, i) => {
+    if (elem) {
+      document.querySelectorAll(".panel price > ul")[i].style.visibility =
+        "visible";
+    } else {
+      document.querySelectorAll(".panel price > ul")[i].style.visibility =
+        "hidden";
+    }
+  }); */
+  let newSum = "";
+  let sum = basePrice;
+
+  for (const element in state) {
+    if (state[element]) {
+      sum += ingredients[element].price;
+      newSum += `<li>${ingredients[element].price} ${ingredients[element].name}</li>`;
+    }
+  }
+  document.querySelector("strong").innerText = `$${sum}`;
+  document.querySelector(".price ul").innerHTML = newSum;
 }
 
-
-renderEverything()
-
+renderEverything();
 // Iteration 1: Example of a click event listener on `<button class="btn btn-pepperonni">`
-document.querySelector('.btn.btn-pepperonni').onclick = function() {
-  state.pepperonni = !state.pepperonni
-  renderEverything()
-}
+document.querySelector(".btn.btn-pepperonni").onclick = function(event) {
+  event.target.classList.toggle("active");
+  state.pepperonni = !state.pepperonni;
+  renderEverything();
+};
 
 // Iteration 1: Add click event listener on `<button class="btn btn-mushrooms">`
-
+document.querySelector(".btn-mushrooms").onclick = function(event) {
+  event.target.classList.toggle("active");
```

Comment:
you can also do this and for every other toggle below in the renderButtons() function which will save a lot of code and makes your code cleaner.